### ISSUE TITLE
Harvesting berries uses held item as soon as it finishes

### ISSUE THREAD

**Issue by MonsterHorns** [anegostudios/VintageStory-Issues#3715]
### Game Version

v1.19.4

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

When harvesting berries from a berry bush by holding right mouse button, as soon as it finishes the held item is immediately used as if clicked with right mouse button again.
The consequences range from mildly annoying (placing torches or blocks, including those that need a tool to pick up again, putting on armor, etc) to infuriating: I spent several ingame days chasing and capturing a wild goat kid, only to accidentally release it in the middle of nowhere because the basket was highlighted when I stopped to gather some berries.

### How to reproduce

1. Put an item with a rightclick action in the hotbar or backpack slot, such as a torch or a reed basket with an animal inside
2. Highlight the item
3. Harvest a berry bush
4. Fail to release the button in time to prevent the item from being used

**Comment by pingouyn** [comment]
I agree this is really annoying.  If there was a way to wait till the mouse click is released before doing any additional action I guess that would solve the issue.
